src/ImageTransformations.jl

```diff
@@ -1,3 +1,40 @@
+"""
+
+This package provides support for image resizing, image rotation, and other spatial transformations of arrays.
+
+- `InvWarpedView`: Create a view of `img` that lazily transforms any given index `I` passed to `wv[I]` so that `wv[I] == img[inv(tinv)(I)]`.
+- `WarpedView`: Create a view of `img` that lazily transforms any given index `I` passed to `wv[I]` so that `wv[I] == img[tform(I)]`.
+- `imresize`: Upsample/downsample the image `img` to a given size `sz` or axes `inds` using interpolations.
+- `restrict`: A much more efficient version of `imresize` that two-folds/down-samples image to approximate 1/2 size. (This is now provided by ImageBase.)
+- `imrotate`: Rotate image `img` by `θ`∈[0,2π) in a clockwise direction around its center point.
+- `warp`: Transform the coordinates of `img`, returning a new `imgw` satisfying `imgw[I] = img[tform(I)]`.
```

Comment:
Maybe this should also appear next to `WarpedView` and explain that it's an eager variant. https://en.wikipedia.org/wiki/Lazy_evaluation